install/oddjob/com.redhat.idm.trust-fetch-domains.in

```diff
@@ -153,42 +159,47 @@ trusted_domain = trusted_domain_entry.single_value.get('cn').lower()
 # At this point if we didn't find trusted forest name, an exception will be raised
 # and script will quit. This is actually intended.
 
-oneway_keytab_name = '/var/lib/sss/keytabs/' + trusted_domain + '.keytab'
-oneway_principal = str('%s$@%s' % (own_trust_flatname, trusted_domain.upper()))
+if not (options.admin and options.password):
+    oneway_keytab_name = '/var/lib/sss/keytabs/' + trusted_domain + '.keytab'
```

Comment:
Does Debian use the same directory or do we need another variable in ipaplatform.path?

``os.path.join('/var/lib/sss/keytabs', '{}.keytab'.format(trusted_domain))``

Reply:
I checked Debian rules and they do not override /var/lib/sss at all. Also, SSSD does not give a way to override /var/lib/sss/keytabs.